client/client.go

```diff
@@ -191,22 +191,34 @@ func loadServerTrust(conf Config) (*x509.CertPool, error) {
 		return nil, nil
 	}
 
-	certs, err := x509.SystemCertPool()
+	syscerts, err := x509.SystemCertPool()
+	// fmt.Println(syscerts)
 	if err != nil {
 		return nil, err
 	}
 
 	// Read certificate file.
-	cacert, err := ioutil.ReadFile(conf.ServerCert)
+	servcert, err := ioutil.ReadFile(conf.ServerCert)
+	// fmt.Println(servcert)
 	if err != nil {
 		return nil, err
 	}
-	certs.AppendCertsFromPEM(cacert)
 
-	if len(certs.Subjects()) == 0 {
+	if servcert == nil && syscerts == nil {
```

Comment:
The intention is to catch when there is neither a specified certificate nor a pool of system certificates. I don't think we have handled the case where the file exists, but is empty. I'm not even sure if that would return nil or not. Might be worth checking.